pkg/sabakan-state-setter/controller.go

```diff
@@ -72,6 +73,10 @@ func NewController(etcdClient *clientv3.Client, sabakanAddress, serfAddress, con
 	if err != nil {
 		return nil, err
 	}
+	sabakanTLSClient, err := newSabakanGQLClient(sabakanAddressHTTPS)
```

Comment:
Please fix `newSabakanGQLClient` and `SabakanClientWrapper` as follows.
The caller should have only one SabakanClientWrapper.

```suggestion
	sabakanClient, err := newSabakanClient(sabakanAddress, sabakanAddressHTTPS)
```

Reply:
I changed `SabakanClientWrapper` to have two sabakan client(HTTP and HTTPS) instead of the Controller having two sabakan client.